pkg/eve/runme.sh

```diff
@@ -92,6 +92,13 @@ docker run --rm lfedge/eve -f qcow2 installer_iso > eve-iso.img
 
 The two raw formats "live" and "installer_raw" support an optional
 last argument specifying the size of the image in Mb.
+
+Optionally you can specify platform:
+
+ -p <platform>
```

Comment:
Should it be `-t`?

Reply:
@rene suggested using PLATFORM in env for make and i use same flag in runme.sh

Reply:
I can see `-t <target>` few lines above, and `-t*)` in switch below. Can you please sync parameters?

Reply:
I plan to do a rebase and fix it, thanks.